app/ide-desktop/lib/client/electron-builder-config.ts

```diff
@@ -109,6 +111,9 @@ export function createElectronBuilderConfig(passedArgs: Arguments): electronBuil
         ],
         mac: {
             // We do not use compression as the build time is huge and file size saving is almost zero.
+            // This type assertion is UNSAFE, and any users MUST verify that
```

Comment:
we have an unsafe type assertion here. it would be good to double-check that this is ok